src/components/map/PlaceDetail.tsx

```diff
@@ -0,0 +1,64 @@
+'use client';
+
+/* eslint-disable react-hooks/exhaustive-deps */
+/* eslint-disable no-param-reassign */
+/* eslint-disable no-return-assign */
+/* eslint-disable no-undef */
+/* eslint-disable no-shadow */
+/* eslint-disable no-unused-vars */
+/* eslint-disable no-new */
+
+import React, { useState, useEffect } from 'react';
+
+interface PlaceDetailsProps {
+  placeId: string;
+}
+
+export default function PlaceDetail({ placeId }: PlaceDetailsProps) {
+  const [place, setPlace] = useState<google.maps.places.PlaceResult | null>(null);
```

Comment:
```suggestion
  const [place, setPlace] = useState<google.maps.places.PlaceResult>(null);
```
null을 기본값으로 설정하고 있어서 제네릭에 null을 제거해도 동일하게 동작할 것 같습니다.